recipes/pyctest/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyctest" %}
+{% set version = "0.0.10" %}
+{% set sha256 = "28c6f154fd4953e72b42441015534a32250b265433628d232acc5befb969eed5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - git
```

Comment:
Why do you need git here?

Reply:
The CMake build system checks if the submodule folders have anything in them and if they haven't been checked out, it does a `git submodule update --recursive <path>`.